### PR TITLE
add branding assets to site

### PR DESCRIPTION
**Pull request by virkt25**
LB4 Branding Assets so we have permanent links for linking in future.

### CODE REVIEW

**Review by shimks — COMMENTED** [review]
Is this preliminary work being done for an upcoming PR? If it is, I personally think it should be done as a part of that PR. If it isn't, I'd still imagine it would have to deal with some kind of PR that would utilize the new assets. Could you link what the related issue is to this is so that we could better judge how these assets are going to be used?

**Comment by virkt25** [comment]
@shimks I guess I should've been more clear. This is not associated with any upcoming PRs.

The purpose of this PR is to add our LB4 assets to the site so we have links we can use to link to as well as later on add them to a branding page (but that might have to wait till the site re-write). @dhmlau needed to reference the assets for the blog post. Out CLI references a file using github's raw content domain but we should switch to the loopback.io domain (another way to promote the project). That was based on a discussion with @raymondfeng -- the switch will happen once this PR lands.

**Review by shimks — APPROVED** [review]

File: images/branding/logo/black/loopback.svg [hunk 1]
```diff
@@ -0,0 +1 @@
..."/><path d="M274.51,52.51h16c5.68,0,9.13,3.5,9.13,8.84s-3,7.21-6.17,7.41v.29c3,0,7.46,2.18,7.46,8,0,5.54-3.81,9.88-8.89,9.88H274.51ZM281,66.88h8.05a3.41,3.41,0,0,0,3.76-3.66V61.69c0-2.32-1.39-3.65-3.76-3.65H281Zm0,14.57h9.09c2.47,0,3.95-1.33,3.95-3.75V76c0-2.42-1.48-3.81-3.95-3.81H281Z" transform="translate(-26.05 -26)"/><path d="M303.55,80c0-5.29,4-7.9,11-7.9H319V70.19c0-2.87-1.43-4.54-4.79-4.54a6.71,6.71,0,0,0-5.87,3.25l-3.76-3.35c1.88-3,4.89-4.94,10.08-4.94,7,0,10.67,3.31,10.67,9.18V81.94h2.56v5h-3.5c-2.72,0-4.4-1.78-4.79-4.44h-.3c-.84,3.31-3.65,5-7.51,5C306.52,87.57,303.55,84.56,303.55,80ZM319,79.28V75.87h-4C311.6,75.87,310,77,310,79v.84c0,2.07,1.43,3.11,3.9,3.11C316.69,83,319,81.7,319,79.28Z" transform="translate(-26.05 -26)"/><path d="M329.78,74c0-8.25,4.3-13.43,11.9-13.43,5.19,0,8.6,2.47,10.13,6.52l-5.19,2.32a4.81,4.81,0,0,0-4.94-3.76c-3.5,0-5.28,2.47-5.28,6.08v4.69c0,3.61,1.78,6,5.28,6,3,0,4.5-1.73,5.44-4L352,80.76c-1.68,4.54-5.29,6.81-10.28,6.81C334.08,87.57,329.78,82.34,329.78,74Z" transform="translate(-26.05 -26)"/><path d="M355.36,50.43h6.33v22h.24l3.41-4.64,5.78-6.62h7.16l-8.89,10L379.22,87h-7.51l-6.62-11.56-3.4,3.71V87h-6.33Z" transform="translate(-26.05 -26)"/></svg>
```
Comment (dhmlau) [inline images/branding/logo/black/loopback.svg:1]: there is missing new line  (and other svg files as well).

**Review by dhmlau — APPROVED** [review]
Thanks for putting it up!!  Just one comment about the missing new lines for the svg files.  Not sure if we need to add it or that's coming from the tool Cody used.

File: images/branding/logo/black/loopback.svg (hunk 1 above)
Comment (virkt25) [reply images/branding/logo/black/loopback.svg:1]: SVG's are rendered as code on GitHub but not when served with the proper `Content-Type` header over Web. IDT we should be worrying about the new lines in `.svg` files.

**Review by b-admike — APPROVED** [review]

**Pull request merged**